Rakefile

```diff
@@ -57,6 +62,20 @@ task 'build:vlckit:ios' do
   end
 end
 
+desc 'Build TVVLCKit'
+task 'build:vlckit:tv' do
+  puts 'Building TVVLCKit'
+
+  plugin_file = 'Resources/MobileVLCKit/vlc-plugins-AppleTV.xcconfig'
+  required_dirs = ['./libvlc/vlc/install-AppleTVSimulator', './libvlc/vlc/build-AppleTVSimulator']
+
+  if File.exist?(plugin_file) && dirs_exist?(required_dirs)
+    puts 'Found pre-existing build directory. Skipping build'
+  else
+    sh "./compileAndBuildVLCKit.sh #{VLC_FLAGS_TV}"
+  end
```

Comment:
I am still not sure if we really need to skip the build here.  I see this as a potential source for a wrong test outcome where we use an outdated build directory. We right now don't need to be fast or save time so our number 1 priority should just be an accurate test.

Reply:
That's true. But if we have the right build directory cached to start off with, I can't really see anything going wrong - since we aren't modifying anything in the build directory. Plus, if anything changes to the way we build VLC, the old cache would be ignored.

I think by caching the build directory, we have one less thing to worry about.

Reply:
What we could also do is keep using the cache for now and remove it once all tests are written. This way we don't have to wait for an hour to build & run tests while submitting minor patches/additions.